text/src/autogluon/text/automm/predictor.py

```diff
@@ -432,6 +433,15 @@ def fit(
         # save artifacts for the current running, except for model checkpoint, which will be saved in _fit()
         self.save(save_path)
 
+        if hasattr(config, MATCHER):
```

Comment:
Raise a warning to say that it is experimental.

Reply:
Done.